src/main/java/org/spideruci/tacoco/analysis/AbstractAnalyzer.java

```diff
@@ -76,6 +83,20 @@ public static AbstractAnalyzer getInstance() {
 		}
 	}
 	
+	public HashSet<String> getIgnoreTestClassSet()
```

Comment:
yup, go ahead and do it, i.e. placing the ignore test method in the AbstractRuntimeAnalyzer! i was actually waiting for the commit and forgot to reply on this thread. Sorry about that.